config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-cluster-api/cluster-api-controller:latest
+      - image: gcr.io/k8s-staging-cluster-api/cluster-api-controller:latest
```

Comment:
This file doesn't exist yet, I'm wondering if we should update this now or hold off.

Eventually, we'd want to have an automated prow job to build and push this image on post-submit.

Reply:
Which file doesn't exist yet?

Reply:
s/file/container image/

Reply:
Given that this is master, I'm ok updating it now. That said, could we change the tag to `master` instead of `latest`?

Reply:
We can change it if/when we need to when we wire the process up.